openc3-cosmos-cmd-tlm-api/app/controllers/tools_controller.rb

```diff
@@ -58,4 +58,18 @@ def importmap
     end
     render :json => result
   end
+
+  def auth
+    url = ENV['OPENC3_KEYCLOAK_EXTERNAL_URL']
+    unless url
+      url = ENV['OPENC3_KEYCLOAK_URL']
+      if url == "http://openc3-keycloak:8080"
+        # Externally should be just /auth
+        url = "/auth"
```

Comment:
This is inside the `unless ENV['OPENC3_KEYCLOAK_EXTERNAL_URL']` so if keycload external url is NOT defined. So why are we talking about externally?

Reply:
The frontend is external to the cluster, so it needs to get a path to keycloak that isn't the internal openc3-keycloak.